release_notes/ocp_3_7_release_notes.adoc

```diff
@@ -755,15 +755,24 @@ $ curl --head https://$route -k
    ...
 ----
 
-[[ocp-37-semi-automatic-namespace-wide-egress-ip]]
-==== Semi-automatic Namespace-wide Egress IP
+[[ocp-37-enabling-static-IPs-for-external-project-traffic]]
+==== Enabling Static IPs for External Project Traffic (Technology Preview)
 
-All outgoing external connections from a project will share a single
-fixed-source IP address and will send all traffic via that IP so that external
-firewalls can recognize the application associated with a packet.
+As a cluster administrator, you can assign specific, static IP addresses to
+projects, so that traffic is externally easily recognizable. This is different
+from the default egress router, which is used to send traffic to specific
+destinations.
+
+Recognizable IP traffic increases cluster security by ensuring the origins
+visible. Once enabled, all outgoing external connections from the specified
```

Comment:
Is that suposed to be "by ensuring the origin *is* visible"? Or did even more text accidentally get deleted there?

Reply:
Oh, apparently that was just copied from managing_networking.adoc. Well, same fix is needed there.